odl/discr/discr_ops.py

```diff
@@ -258,13 +260,12 @@ def __init__(self, domain, range=None, ran_shp=None, **kwargs):
          [5.0, 6.0, 7.0, 8.0],
          [1.0, 2.0, 3.0, 4.0]]
         """
+        from builtins import range as builtin_range
```

Comment:
Hm, I rather see this as an "emergency import" and would prefer not polluting the module namespace with this. Where possible, only `range` should be available. 
That's not a strong preference, though, for me it's just a second case beside an expensive import where lazy imports are okay.